### PR TITLE
Shalamai.week3

### PR DESCRIPTION
**Pull request by shalamai**
please check

---
This change is [Reviewable](https://reviewable.io/reviews/presly808/aco17/18)

### CODE REVIEW

**Comment by shalamai** [comment]
please check

File: VladShalamai/src/travelagency/Run.java [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+ */
+public class Run {
+
+    public static void main(String[] args) {
+
+        DataBase dataBase = new DataBase();
+        UserController user = new UserController(dataBase);
+        AdminController admin = new AdminController(dataBase);
+
+        for (int i = 0; i < 5; i++) {
```
Comment (presly808) [inline VladShalamai/src/travelagency/Run.java]: create an init data method.

File: VladShalamai/src/travelagency/controller/AdminController.java [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+/**
+ * Created by Влад on 28.11.2016.
+ */
+public class AdminController {
+
+    DataBase dataBase;
```
Comment (presly808) [inline VladShalamai/src/travelagency/controller/AdminController.java]: add access modifiers

File: VladShalamai/src/travelagency/controller/AdminController.java [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+    Pattern pattern = Pattern.compile("[^a-zA-Z\\s]");
+    private static int password = 12345;
+
+    public AdminController(DataBase dataBase) {
+        this.dataBase = dataBase;
+    }
+
+    public static int getPassword() {
```
Comment (presly808) [inline VladShalamai/src/travelagency/controller/AdminController.java]: for modifying use next signature of method 
checkPass(int pass)

File: VladShalamai/src/travelagency/controller/AdminController.java [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+    public void addTour(String name, long price, MyDate startDate, MyDate endDate, String transport, Hotel hotel) {
+
+        Matcher matcherName = pattern.matcher(name);
+        Matcher matcherTransport = pattern.matcher(transport);
+
+        if (matcherName.find() || matcherTransport.find() || price < 0 || startDate.equals(endDate)) {
+            return;
+        }
+
+        dataBase.getTours().add(new Tour(name, price * 100, startDate, endDate, transport, hotel));
```
Comment (presly808) [inline VladShalamai/src/travelagency/controller/AdminController.java]: set name to the value '100'

File: VladShalamai/src/travelagency/controller/AdminController.java [hunk 5]
```diff
@@ -0,0 +1,85 @@
...
+
+    }
+
```
Comment (presly808) [inline VladShalamai/src/travelagency/controller/AdminController.java]: add javadoc for understanding

File: VladShalamai/src/travelagency/controller/UserController.java [hunk 6]
```diff
@@ -0,0 +1,94 @@
...
+        for (Tour tour : dataBase.getTours()) {
+            sb.append(tour.toString());
+        }
+
+        return sb.toString();
+    }
+
+    public void sendTourRequest(int id, String name, String phone, String email) {
+
+        Matcher matcherName = patternName.matcher(name);
```
Comment (presly808) [inline VladShalamai/src/travelagency/controller/UserController.java]: validator.validate(name)

File: VladShalamai/src/travelagency/db/DataBase.java [hunk 7]
```diff
@@ -0,0 +1,33 @@
...
+
+import java.util.ArrayList;
+
+
+/**
+ * Created by Влад on 26.11.2016.
+ */
+public class DataBase {
+
+    private ArrayList<Tour> tours = new ArrayList<>();
```
Comment (presly808) [inline VladShalamai/src/travelagency/db/DataBase.java]: List<Tour> tours = new ArrayList<>();

File: VladShalamai/src/travelagency/model/Address.java [hunk 8]
```diff
@@ -0,0 +1,51 @@
+package travelagency.model;
+
+/**
+ * Created by Влад on 26.11.2016.
+ */
+public class Address {
```
Comment (presly808) [inline VladShalamai/src/travelagency/model/Address.java]: override toString and equals methods

**Pull request closed without merging**